### DISCUSSION TITLE
Run / Install on Ubuntu Linux?

### DISCUSSION THREAD

**Discussion by ctsmith2** [JetSetIlly/Gopher2600#34] [Q&A]
Is it possible to Run / Install on Ubuntu Linux?

**Comment by JetSetIlly** [comment]
Yes. It should be fine. I compiled the linux binary on a Debian system so the dependencies should be the same as your machine. You might need to install some things with the package manager (eg. SDL2) but I would imagine you have them installed already.

If you have any problems let me know and we can think it through.

The other thing to note is that once you've downloaded the binary from the release page you'll need to set the executable bit on the binary with `chmod u+x <download_path>/gopher2600_linux_amd64`
